cppapi/client/devapi_base.cpp

```diff
@@ -1212,6 +1212,7 @@ void Connection::set_timeout_millis(int millisecs)
 }
 
 
+//TODO all command_inout methods must be moved to DeviceProxy
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this "TODO" comment. ([cpp:S1135](https://sonarcloud.io/organizations/tango-controls/rules#rule_key=cpp%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.tango-controls%3Acpp-tango%3Atango-9-lts&pullRequest=477&issues=AWWPm0Fu3Qu1svVwsabC&open=AWWPm0Fu3Qu1svVwsabC)

[ISSUE_KEY]: AWWPm0Fu3Qu1svVwsabC